server/models/assessRisksAndNeeds/riskSummary.ts

```diff
@@ -1,10 +1,12 @@
 export type RiskScore = 'LOW' | 'MEDIUM' | 'HIGH' | 'VERY_HIGH'
 
 export default interface RiskSummary {
-  whoIsAtRisk?: string | null
-  natureOfRisk?: string | null
-  riskImminence?: string | null
-  riskIncreaseFactors?: string | null
-  riskMitigationFactors?: string | null
-  riskInCommunity: Partial<Record<RiskScore, string[]>>
+  summary: {
```

Comment:
I left a comment about this on your other PR,  but it should have come in this one - should this interface now be just `Risk(s?)` rather than `RiskSummary` if it has a `summary` field on it?